frontend/rockcraft.yaml

```
@@ -11,7 +10,7 @@ platforms:
     amd64:
```

Add new non-root user:
```suggestion
    amd64:
run-user: _daemon_
```